crates/sui-package-alt/src/sui_flavor.rs

```diff
@@ -23,29 +23,36 @@ pub struct SuiFlavor;
 impl MoveFlavor for SuiFlavor {
     type FlavorDependency<P: ?Sized> = OnChainDependency;
 
+    fn name() -> String {
```

Comment:
Probably we want to rename this.
```suggestion
    fn flavor() -> String {
```